### PR TITLE
Adding more context to readme

### PR DESCRIPTION
**Pull request by basiliskus**
# Description

- Added instructions on how to test new assertions from a new branch/PR
- Added more context on how to run tests

## Issue

#1540

### CODE REVIEW

File: rs-e2e/README.md [hunk 1]
```diff
@@ -13,14 +13,44 @@ Information on how to set up the sample files evaluated by the tests can be foun
...
+- [Automated Staging Test - Run integration tests](https://github.com/CDCgov/trusted-intermediary/actions/workflows/automated-staging-test-run.yml) triggers a couple of hours later and runs the Automated Tests on the input files in `/examples/Test/Automated` and the output files in the Azure storage container.
+
+### Manually from your local machine
+
+When running locally, we usually run the tests from either the command line using gradle, or from IntelliJ. Please note that even though you are running the test from your local machine, the test will need to connect to the Azure container to pull the output files to apply the assertions on. For this reason, you will need to set the `AZURE_STORAGE_CONNECTION_STRING` environment variable to authenticate the connection to Azure. You can find the value for `AZURE_STORAGE_CONNECTION_STRING` in Keybase (keybase://team/cdc_ti/service_keys/TI/staging/azure-storage-connection-string-for-automated-rs-e2e-tests.txt).
+
+From the command line:
+
+1. Set the `AZURE_STORAGE_CONNECTION_STRING` environment variable in your shell
```
Comment (luis-pabon-tf) [inline rs-e2e/README.md:29]: The only thing this is missing to me is that nothing says what this env var should be set to anymore.

**Review by luis-pabon-tf — CHANGES_REQUESTED** [review]
A big improvement over the original doc, only had one suggestion

File: rs-e2e/README.md (hunk 1 above)
Comment (basiliskus) [reply rs-e2e/README.md:29]: It is in the paragraph above

Comment (luis-pabon-tf) [reply rs-e2e/README.md:29]: Oh, I should've checked the full file and not the div. Well the PR is all good then! 👍

**Review by luis-pabon-tf — APPROVED** [review]

File: rs-e2e/README.md [hunk 2]
```diff
@@ -13,14 +13,44 @@ Information on how to set up the sample files evaluated by the tests can be foun
...
-  - Run the [automated-staging-test-run](/.github/workflows/automated-staging-test-run.yml) action
-- Locally
-  - Set the `AZURE_STORAGE_CONNECTION_STRING` environment variable to the [value in Keybase](keybase://team/cdc_ti/service_keys/TI/staging/azure-storage-connection-string-for-automated-rs-e2e-tests.txt)
-  - Run the tests with `./gradlew rs-e2e:clean rs-e2e:automatedTest`
+### Automatically
+
+There are two scheduled tasks that run every weekday around midnight EST:
+
+- [Automated Staging Test - Submit Messages](https://github.com/CDCgov/trusted-intermediary/actions/workflows/automated-staging-test-submit.yml) submits the messages in `/examples/Test/Automated`
+- [Automated Staging Test - Run integration tests](https://github.com/CDCgov/trusted-intermediary/actions/workflows/automated-staging-test-run.yml) triggers a couple of hours later and runs the Automated Tests on the input files in `/examples/Test/Automated` and the output files in the Azure storage container.
```
Comment (pluckyswan) [inline rs-e2e/README.md:21]: Is there a specific storage container? If not, LGTM!

Comment (basiliskus) [reply rs-e2e/README.md:21]: I can add the name

Comment (basiliskus) [reply rs-e2e/README.md]: Added

**Pull request merged**